docs/python/tutorial-fastapi.md

```diff
@@ -0,0 +1,660 @@
+---
+Order: 10
+Area: python
+TOCTitle: FastAPI Tutorial
+ContentId: 0d32bced91aa5c2ee5696fc7995370ae
+PageTitle: Python and FastAPI Tutorial in Visual Studio Code
+DateApproved:
+MetaDescription: Python FastAPI tutorial showing IntelliSense and debugging support in Visual Studio Code, the best Python IDE.
```

Comment:
True but we should be humble

Reply:
I copied this from the [Django](https://github.com/microsoft/vscode-docs/blob/main/docs/python/tutorial-django.md) and [Flask ](https://github.com/microsoft/vscode-docs/blob/main/docs/python/tutorial-flask.md) tutorials FWIW 😅 I think it might have been added for SEO ranking?

Reply:
Too funny, I never noticed. No one has complained and you may be right about SEO juice. Leave it in.